### PR TITLE
Front Page: Add scastie embedding

### PR DESCRIPTION
**Pull request by MasseGuillaume**
2017-09-18-171252_1920x1080_scrot

### CODE REVIEW

**Comment by MasseGuillaume** [comment]
It will randomly select a snippet from this list:

https://scastie.scala-lang.org/liEaE8NoSlqYhcmD1yvgUw
https://scastie.scala-lang.org/mcmef1f6TMCX8PQZm5X0Tw
https://scastie.scala-lang.org/R1txDbFnSwKzxVJ39ffgYw
https://scastie.scala-lang.org/yzZaG0TkTPasRnYvYNNLcw
https://scastie.scala-lang.org/64UAY3UrRNyeo1XVDV5j0Q
https://scastie.scala-lang.org/efJw6gtqQKu5AC75B0LNVA
https://scastie.scala-lang.org/CtQirsr6QjW0R4v3VkzJVA
https://scastie.scala-lang.org/BqYOO4d9SnSv8lVpNZgftw
https://scastie.scala-lang.org/CGHJ8X2vQQSJshHyplcPHA
https://scastie.scala-lang.org/NcTSnSx7T3qeoiTlEkn0cw
https://scastie.scala-lang.org/pGUX2rbRT96ZnSWYQxeUNg
https://scastie.scala-lang.org/3e166cHFR4qeQXuwSZh5EA
https://scastie.scala-lang.org/QU48GWJATH6cctgVwVcsmQ
https://scastie.scala-lang.org/sTbtRfGyQNSCSy108HiJzA
https://scastie.scala-lang.org/CM5gMYT4SkyPmWGX2whZyQ
https://scastie.scala-lang.org/FgnyudFsT7CRuFLz7wRD1Q
https://scastie.scala-lang.org/UBY0uNu6S0Kor5lcQSjrjg
https://scastie.scala-lang.org/bKjfsVlSSjC97bczClvNjA
https://scastie.scala-lang.org/ZIEdUV1ZT32sQe3aRP7Bug
https://scastie.scala-lang.org/AMXnCpvsSlq23834HJSrNg

Based on: https://contributors.scala-lang.org/t/contest-scala-lang-org-frontpage-code-snippet/1141/39

File: resources/js/functions.js [hunk 1]
```diff
@@ -196,53 +196,37 @@ $(document).ready(function() {
...
+    "pGUX2rbRT96ZnSWYQxeUNg",
+    "3e166cHFR4qeQXuwSZh5EA",
+    "QU48GWJATH6cctgVwVcsmQ",
+    "sTbtRfGyQNSCSy108HiJzA",
+    "CM5gMYT4SkyPmWGX2whZyQ",
+    "FgnyudFsT7CRuFLz7wRD1Q",
+    "UBY0uNu6S0Kor5lcQSjrjg",
+    "bKjfsVlSSjC97bczClvNjA",
+    "ZIEdUV1ZT32sQe3aRP7Bug",
+    "AMXnCpvsSlq23834HJSrNg"
```
Comment (julienrf) [inline resources/js/functions.js]: I’m a bit scared of putting all these snippets on scala-lang. They are interesting pieces of code but a majority of them is a bit hard to understand. Also, you should credit the authors.

**Comment by MasseGuillaume** [comment]
https://scastie.scala-lang.org/dX96ma1ZSmOnFcAgr2Mogw
https://scastie.scala-lang.org/WdXKPq6kTvqtQe4YIjDUYA
https://scastie.scala-lang.org/eUqfuX9wTdaZwetCbCa1aw
https://scastie.scala-lang.org/0vD0equiRVafkBIeSxJ3eQ
https://scastie.scala-lang.org/00ORgmZsRqaPLsllyhA8SQ
https://scastie.scala-lang.org/dOrqJ7o7Q06dm37Zmb7vKg
https://scastie.scala-lang.org/qpBNSiv2Rgi1rHxbkRhz0g
https://scastie.scala-lang.org/P3Qm9rH5TpSqxKHUrSmHdA
https://scastie.scala-lang.org/NlsWGM66QoO7Ga11uKp5pQ
https://scastie.scala-lang.org/oDL8ZqmnReyX15266JGmog
https://scastie.scala-lang.org/WILnmgGfQOyUeX7nURxgPQ
https://scastie.scala-lang.org/rWl872iZTSekBmx8vJHD1w
https://scastie.scala-lang.org/Blpswq70SuCA0UsHLWdHow
https://scastie.scala-lang.org/lNLxCZy3TuujhHZR5PknhA
https://scastie.scala-lang.org/ppato6k7RMC4KbYZtyiDIg
https://scastie.scala-lang.org/avth1cdCQI6mcgSUXj94kA
https://scastie.scala-lang.org/bWcglkdZSyyzuIv4oLDMig
https://scastie.scala-lang.org/s0oOxZbMSNqQTFkLknhLrQ
https://scastie.scala-lang.org/MsldP2eISWqZYJ0earpo5A

**Comment by MasseGuillaume** [comment]
Scastie embedding on the scala-lang frontpage

**Comment by heathermiller** [comment]
I don't think we should randomly select one of every snippet submitted. I really think we should select a subset of snippets that really serve to show off clear, simple Scala idioms. Some of these are a bit complex, and I would hesitate to show a few of these to beginners.

**Comment by julienrf** [comment]
Here are the snippets that I think are appropriate for our front page:

https://scastie.scala-lang.org/julienrf/B5mv7WHITGGBaOfHFVFPKQ
https://scastie.scala-lang.org/dOrqJ7o7Q06dm37Zmb7vKg
https://scastie.scala-lang.org/ppato6k7RMC4KbYZtyiDIg
https://scastie.scala-lang.org/0vD0equiRVafkBIeSxJ3eQ

I think the others are too complex for newcomers.

**Pull request closed without merging**